vNext/extensions/applicationinsights-react-native/package.json

```diff
@@ -10,6 +10,7 @@
         "url": "https://github.com/microsoft/ApplicationInsights-JS/tree/master/vNext/extensions/applicationinsights-react-native"
     },
     "scripts": {
+        "prepublishOnly": "npm run build",
```

Comment:
Is this new script being used somewhere?

Reply:
on npm publish, so that we no longer need to rely on rush to build it when publishing